### PR TITLE
games-roguelike/tomenet: version bump, 4.7.3

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

File: games-roguelike/tomenet/tomenet-4.7.3.ebuild [hunk 1]
```diff
@@ -35,42 +36,44 @@ PATCHES=(
 
 src_prepare() {
 	default
-	use server || use dedicated || { rm -r ../lib/{config,data,save} || die ;}
+	use server || { rm -r ../lib/{config,data,save} || die ;}
```
Comment (SoapGentoo) [inline games-roguelike/tomenet/tomenet-4.7.3.ebuild:39]: this is a terrible construct, turn it into a normal `if ! use server; then ... fi` construct

File: games-roguelike/tomenet/tomenet-4.7.3.ebuild [hunk 2]
```diff
@@ -80,20 +83,10 @@ src_install() {
 	doins ../.tomenetrc
 }
 
-pkg_preinst() {
-	gnome2_icon_savelist
-}
-
 pkg_postinst() {
```
Comment (SoapGentoo) [inline games-roguelike/tomenet/tomenet-4.7.3.ebuild:86]: `xdg_postinst` won't get called here

File: games-roguelike/tomenet/files/tomenet-4.7.3-makefile.patch [hunk 3]
```diff
@@ -0,0 +1,32 @@
+diff --git a/makefile b/makefile
+index ee56a33..7f3c81f 100644
+--- a/makefile
++++ b/makefile
```
Comment (SoapGentoo) [inline games-roguelike/tomenet/files/tomenet-4.7.3-makefile.patch:4]: can you just check whether the makefile also respects `CPPFLAGS`? This is a user flag that should be respected

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: games-roguelike/tomenet/files/tomenet-4.7.3-makefile.patch (hunk 3 above)
Comment (DarthGandalf) [reply games-roguelike/tomenet/files/tomenet-4.7.3-makefile.patch:4]: Done

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**